pkgs/development/libraries/science/electronics/coloquinte/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, cmake, lemon-graph, eigen, boost
+}:
+
+stdenv.mkDerivation rec {
+  pname = "coloquinte";
+  version = "0.3.1";
+
+  meta = with lib; {
+    description = "Placement library for electronic circuits";
+    homepage    = "https://github.com/Coloquinte/PlaceRoute";
+    license     = licenses.mit;
+    platforms   = platforms.linux;
+    maintainers = with lib.maintainers; [ coloquinte ];
+  };
```

Comment:
We usually put the meta attribute at the very bottom of the derivation. See how other derivations are built.